### ISSUE TITLE
Movies not visible

### ISSUE THREAD

**Issue by draciti** [WormBase/website#5545]
2 examples:

on the gene expression page:

**Comment by draciti** [comment]
Just checking in on this issue.
thanks
d

**Comment by sibyl229** [comment]
Hi @draciti sorry about the lateness. It seems that the encodings of the videos are not compatible with web display.

For the Expr pattern videos, since they stored on our server, I think we can re-encode them. Do you mind helping me verify the resolution is still okay? Is there any particular video that is good for testing?

For the RNAi videos, it looks like they are from another site. This would be a little trickier. Do you think we can ask them to change the encoding of their videos?

Thanks

**Comment by draciti** [comment]
hi sybil, the expression pattern movies looks fine thanks!

for the RNAi movies: they come from the Piano lab.
We originally wanted to store the movies on our machine - I checked the records in the Movie wiki and there is this note:
'for the Piano Movies (stored at the rnai.org) database we have tried to obtain -and failed to have- the access to full movies from Kris Gunsalus at NYU. For this reason we requested a "database info-DB_INFO" tag to store the urls and link out directly - October 2013 DR'

I went now to rnai.org, the site where the movies are stored, but it seems to be down. I sent an e-mail to the administrator to understand if it is defunct or just asleep :) 
I'll keep you posted. thx

**Comment by draciti** [comment]
Kris just replied, I'll ask him again if we can get the files or if they can convert them.
'Hi Daniela,
Really sorry about that. We had a server crash recently and I haven’t had time to get around to fixing it yet. ;-( I’ll try to do it next week.

I just tried Phenobank and that seems to be down too… odd. Maybe try WormBase for now. They used to have links to movies.

Best regards,
Kris'

**Comment by draciti** [comment]
for the records:
Hi Kris, thanks for your prompt reply-

In fact, I am from WormBase and I was trying to connect to the rnai.org site as the movies are no longer compatible with our web display. Our developer asked if it was possible for you to re-encode the movies, or if we could download the original files and store them locally.
Let me know what would work for you, we think that your collection is a great addition to WormBase and would like to still have viable links.

Many thanks,

Daniela

**Comment by draciti** [comment]
Thanks Daniela.

OK, I will look into it. I would need to check with Fabio regarding the movies. I personally wouldn’t mind letting you store them.

We recently discovered that a bunch of our other old Quicktime movies also needed to be converted. I’ll look into that.

Best regards,
Kris

**Comment by draciti** [comment]
Sybil, besides the Piano collection we have other RNAi movies that are not working, for example http://www.wormbase.org/species/c_elegans/rnai/WBRNAi00081283#3--10

can you re-encode those? I am testing the Datomic widget
thanks

**Comment by sibyl229** [comment]
Hi @draciti , I have copies of the videos on the staging server. But I imagine the staging server isn't the primary repository where you add the videos, is it? And I imagine the primary repository for videos is synced with staging server each release. 

In that case, it might be better to re-encode the videos in the primary repository, so the encoding is done once for all. If I were to re-encode the copies on staging, the re-encoding copies will be overwritten by a sync. Does that make sense?

**Comment by draciti** [comment]
that's correct @sibyl229 , the primary repo for movies is canopus on this directory
/home/daniela/OICR/Movies
I have no idea on how to re-encode the movies though..
do you have access to canopus?

**Comment by sibyl229** [comment]
@draciti I guess I can re-encode the movies and upload them onto one of the WormBase servers, then you to get and incorporate them into the primary repo. And please remember to backup the originals on canopus before overwriting them :)

**Comment by draciti** [comment]
that will work, thanks! 
Yes, I will back up the originals :)

**Comment by sibyl229** [comment]
Hi @draciti this video
WBPaper00004811/015.G05.i2.z6.mov seems corrupted. I can't open it.

**Comment by sibyl229** [comment]
just a note for myself, the Handbrake preset used for the conversion:
{
  "VersionMajor" : 23,
  "PresetList" : [
    {
      "PictureForceWidth" : 0,
      "PictureHeight" : 0,
      "PictureDeblockCustom" : "qp=0:mode=2",
      "SubtitleAddForeignAudioSubtitle" : false,
      "PictureDeinterlacePreset" : "default",
      "AudioEncoderFallback" : "ca_aac",
      "VideoColorMatrixCode" : 0,
      "VideoOptionExtra" : "",
      "VideoTwoPass" : true,
      "ChildrenArray" : [

      ],
      "PictureDenoiseFilter" : "off",
      "VideoScaler" : "swscale",
      "PresetDescription" : "",
      "PictureDenoiseTune" : "none",
      "VideoQualitySlider" : 22,
      "Mp4iPodCompatible" : false,
      "PictureItuPAR" : false,
      "PictureDeinterlaceCustom" : "",
      "PictureDeinterlaceFilter" : "decomb",
      "FileFormat" : "av_mp4",
      "PictureDenoisePreset" : "light",
      "VideoQualityType" : 2,
      "PictureKeepRatio" : true,
      "PictureForceHeight" : 0,
      "PresetName" : "WBExp2",
      "SubtitleAddCC" : false,
      "VideoLevel" : "4.0",
      "VideoTurboTwoPass" : true,
      "PictureCombDetectCustom" : "",
      "AudioTrackSelectionBehavior" : "first",
      "FolderOpen" : false,
      "PictureLeftCrop" : 0,
      "VideoPreset" : "fast",
      "AudioLanguageList" : [
        "eng"
      ],
      "SubtitleBurnBehavior" : "foreign",
      "PictureDARWidth" : 0,
      "Default" : false,
      "PictureDenoiseCustom" : "",
      "VideoGrayScale" : false,
      "VideoFramerate" : "30",
      "PictureDetelecineCustom" : "",
      "Folder" : false,
      "AudioCopyMask" : [
        "copy:aac"
      ],
      "PicturePARHeight" : 1,
      "PicturePARWidth" : 1,
      "SubtitleAddForeignAudioSearch" : true,
      "SubtitleTrackSelectionBehavior" : "none",
      "VideoQSVDecode" : false,
      "PictureDeblock" : 0,
      "Mp4HttpOptimize" : true,
      "x264UseAdvancedOptions" : false,
      "AudioSecondaryEncoderMode" : true,
      "Type" : 1,
      "VideoEncoder" : "x264",
      "PictureBottomCrop" : 0,
      "VideoAvgBitrate" : 6000,
      "PictureModulus" : 2,
      "SubtitleBurnBDSub" : true,
      "PictureRotate" : "angle=0:hflip=0",
      "VideoFramerateMode" : "pfr",
      "PictureDetelecine" : "off",
      "VideoProfile" : "main",
      "PictureTopCrop" : 0,
      "PictureLooseCrop" : false,
      "PictureRightCrop" : 0,
      "PictureWidth" : 0,
      "ChapterMarkers" : true,
      "PictureCombDetectPreset" : "default",
      "UsesPictureSettings" : 2,
      "PicturePAR" : "off",
      "VideoQSVAsyncDepth" : 4,
      "SubtitleBurnDVDSub" : true,
      "SubtitleLanguageList" : [

      ],
      "PictureAutoCrop" : false,
      "VideoTune" : "",
      "AudioList" : [
        {
          "AudioDitherMethod" : "auto",
          "AudioBitrate" : 160,
          "AudioTrackGainSlider" : 0,
          "AudioEncoder" : "ca_aac",
          "AudioNormalizeMixLevel" : false,
          "AudioTrackDRCSlider" : 0,
          "AudioSamplerate" : "auto",
          "AudioMixdown" : "dpl2",
          "AudioTrackQuality" : -1,
          "AudioCompressionLevel" : -1,
          "AudioTrackQualityEnable" : false
        }
      ],
      "x264Option" : "",
      "UsesPictureFilters" : true
    }
  ],
  "VersionMinor" : 0,
  "VersionMicro" : 0
}

**Comment by draciti** [comment]
Hi @sibyl229, movies are backed up. Let me know how we should proceed.

I confirm this file seems corrupted, I cannot open it as well. 
WBPaper00004811/015.G05.i2.z6.mov

thanks!

**Comment by sibyl229** [comment]
Hi @draciti movie should be fixed now. Could you take a look? Those movies hosted by WB can be played on the site. Those hosted by RNAi.org we provide a link for downloading the movies in the meanwhile, before they fix the encoding. 
https://staging.wormbase.org/species/c_elegans/rnai/WBRNAi00081283#35--10
https://staging.wormbase.org/species/c_elegans/rnai/WBRNAi00000145#35--10
https://staging.wormbase.org/species/all/expr_pattern/Expr5497#03--10

Please remember to include these re-encoded movies I sent you in your primary repo for movies. 

Thanks!

**Comment by sibyl229** [comment]
Hi @draciti , I re-sent you the movies today via email.

Please contact me after RNAi DB has re-encoded the movies. I will need to change the movie links to an embedded video player. 

Thanks
